phantom_connector.py

```diff
@@ -534,60 +559,82 @@ def _find_artifacts(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
         limit_search = param.get("limit_search", False)
         container_ids = param.get("container_ids", "current")
-        values = param.get('values', '')
+        values = param.get("values", "")
+        max_results = param.get("max_results", ARTIFACT_DEFAULT_MAX_RESULTS)
         if limit_search:
             container_ids = list(
```

Comment:
Consider
```
            container_ids = list(
                {
                    a
                    for a in [
                        int(z) if isinstance(z, int) or z.isdigit() else None
                        for z in [
                            self.get_container_id() if y == "current" else y
                            for y in [x.strip() for x in container_ids.replace(",", " ").split()]
                        ]
                    ]
                    if a
                }
            )
```

Reply:
Thanks for review comment, based on the priority - I am merging this PR, as changing the code will require testing and other efforts.